cli.js

```diff
@@ -31,6 +33,19 @@ function run() {
       process.cwd(),
     ], {stdio: 'inherit'});
     break;
+  case 'new-library':
+    if (args[1]) {
+      newLibrary(
+        process.cwd(),
+        args[1]
+      );
+    } else {
+      console.error(
+        'Usage: react-native new-library <LibraryName>'
```

Comment:
So, there are multiple things we want to support in the future.

Type:
- Library that is just exposing js functions. We call it API in the docs but I want to find a better name
- Components

Platform:
- ios
- android

Given those, what do you think the command api should look like?

```
react-native new-library api ios name
react-native new-library component android name
react-native new-library name --type library --platform ios
```

?